crates/rpc/rpc-testing-util/src/trace.rs

```diff
@@ -319,4 +320,29 @@ mod tests {
         println!("Total successes: {}", successes);
         println!("Total failures: {}", failures);
     }
+
+    #[tokio::test]
```

Comment:
we want to ignore this for now, but is useful for manual testing
```suggestion
    #[tokio::test]
    #[ignore]
```